src/stages/models/manager.ts

```diff
@@ -135,6 +149,11 @@ export class StageManager {
 
   @withSpanAsyncV4
   public async getStageById(stageId: string, includeTasks?: boolean): Promise<StageModel> {
```

Comment:
you can add jobId after fetching the stage

Reply:
done